Create a Log Integration Update (#21402)
DataDog/documentation: content/en/developers/integrations/log_integration.md

## Review and deploy the integration

Datadog reviews the log integration based on the guidelines and requirements documented on this page and provides feedback to the Technology Partner through GitHub. In turn, the Technology Partner reviews and makes changes accordingly.

To start a review process, export your log pipeline and relevant custom facets using the **Export** icon on the [Logs Configuration page][3]. 

{{< img src="developers/integrations/export_pipeline.png" alt="Click the Export Pipeline icon to export your log pipeline in Datadog" width="50%">}}

Include sample raw logs with all the attributes you expect to be sent into Datadog by your integration. Raw logs comprise of the raw messages generated directly from the source before they have been ingested by Datadog.

Exporting your log pipeline includes two YAML files:

- One with the log pipeline, which includes custom facets, attribute remappers, and grok parsers.

> - One with the raw example logs with an empty result. Depending on your browser, you may need to adjust your settings to allow file downloads.

Reviewer: ```suggestion
- One with the raw example logs with an empty result. 

Note: Depending on your browser, you may need to adjust your settings to allow file downloads.
```

Once you've downloaded these files, navigate to your [integration's pull request][22] on GitHub and add them in the **Assets** > **Logs** directory. If a Logs folder does not exist yet, you can create one.

> Validations are run automatically in your pull request. A common log pipeline validation error is not populating the `id` field in both YAML files. Ensure that the `id` field matches the `app_id` field in your integration's `manifest.json` file to connect your pipeline to your integration. 

Reviewer: ```suggestion
Validations are run automatically in your pull request. 

Two common validation errors are:
1. The `id` field in both YAML files - Ensure that the `id` field matches the `app_id` field in your integration's `manifest.json` file to connect your pipeline to your integration. 
2. Not providing the result of running the raw logs you provided against your pipeline. If the resulting output from the validation is accurate, take that output and add it to the `result` field in the YAML file containing the raw example logs.
```

> Another common validation error is not providing the result of running the raw logs you provided against your pipeline. If the resulting output from the validation is accurate, take that output and add it to the `result` field in the YAML file containing the raw example logs.

Reviewer: ```suggestion
```
